app/views/audit_mailer/summary.html.erb

```diff
@@ -0,0 +1,31 @@
+<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
+<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> Audit summary").html_safe %></h2>
+<ul style="list-style-type: none;">
+  <% if @audits.size > 0 %>
+    <%= n_('Displaying %{num_audits} of %{total_audits} audits',
+           'Displaying %{num_audits} of %{total_audits} audits', @limit) % {:num_audits => @limit, :total_audits => @count} %>
+    <% @audits.each do |audit| %>
+      <li style="margin: 5px 15px; border-bottom: 1px solid #ccc; padding-bottom: 10px; width: 70%; font-size: 120%;">
+        <%
+           login = audit.user.login rescue nil
```

Comment:
yeah, I am familiar with ```try``` method.
The problem: Audit has it's own unique way to detect the user. audit.user may be a User object, or it may be a String (when no User object). When will try audit.user.try(:login) and the user is a String - we'll still get NoMethodError.
See example at http://fpaste.org/221536/52891214/